data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,25 +1417,59 @@ mission "Hai Reveal [A14] A Time of Trouble"
 		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
 		conversation
 			`Upon docking at <planet>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
-			`	Sayari is intercepted by a Hai functionary and led away as you reach Xilin and the other diplomats, also joined by Danforth. Xilin looks at you all. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. The Navy has blockaded the wormhole to the best of its ability, but the reality is that the Navy is already heavily committed, and barely look to be a match for the Hai fleet numbers. The Hai, on the other hand, do not yet appear to be committed heavily." He looks at Alondo briefly. "We had the unstated privilege of observing the Hai security fleet left behind on Everhope in addition to what we saw beyond the wormhole on our way in."`
+			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as the cabinet is blocking any progress.`
```

Comment:
```suggestion
			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as Cabinet is blocking any progress.`
```

Reply:
I just changed "the Cabinet" and "the cabinet" to "Cabinet" several places. It seems quite odd though, to have a group that is simply called "Cabinet."

Reply:
I've changed it to "the Cabinet" everywhere, since that seems more natural, and matches what I've heard in the real world.

Reply:
It's the proper nown for a group of ministers in countries like the UK and NZ, it's a synonym for government. I was going to comment about whether any Americans would understand what it means.

Reply:
I assumed it was based on the Australian government which calls it "the Cabinet."

Reply:
It's like saying Congress; "Who in Congress is blocking it?" "Who was just elected to Congress?"

Reply:
I'll leave this conversation unresolved hoping for feedback from people more in-the-know about Free Worlds political titles than us.

Reply:
I'm pretty sure nobody knows what is what right now!

Reply:
I changed all references back to Cabinet without "the." Nobody is in "the cabinet" or "cabinet" now; those errors are corrected. Removing "the " takes out four characters.